src/lib.rs

```diff
@@ -980,124 +962,114 @@ impl ConfigFile {
     if self.json.vendor == Some(true) {
       exclude.push("vendor".to_string());
     }
+    Ok(exclude)
+  }
 
-    let raw_files_config = SerializedFilesConfig {
-      exclude,
-      ..Default::default()
+  pub fn to_bench_config(&self) -> Result<Option<BenchConfig>, AnyError> {
+    let mut exclude_patterns = self.resolve_exclude_patterns()?;
+    let serialized = match self.json.bench.clone() {
+      Some(config) => {
+        let mut serialized: SerializedBenchConfig =
+          serde_json::from_value(config)
+            .context("Failed to parse \"bench\" configuration")?;
+        // top level excludes at the start because they're lower priority
```

Comment:
This will make sense in an upcoming PR where it will step through the excludes from back to front and have the ability to opt-out of an exclude via a negated pattern.